demo_extractors/complex/complex.py

```diff
@@ -50,7 +50,7 @@ def run(self, stream: BytesIO, matches: List[yara.Match]) -> Optional[model.Extr
         other = complex_utils.getdata()["result"]
         self.logger.debug("got data from lib")
         # example - accessing yara strings
-        strings = {y[2].decode("utf8") for x in matches for y in x.strings}
+        strings = {z.plaintext().decode("utf8") for x in matches for y in x.strings for z in y.instances}
```

Comment:
I think if you change this to a `sorted([])` the test output would become deterministic and easier to test.